### PR TITLE
fix postsubmit timeout, add release-1.0 config

### PR DESCRIPTION
**Pull request by rkpagadala**
fix postsubmit timeout, add release-1.0 config

### CODE REVIEW

File: prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml [hunk 1]
```diff
@@ -84,4 +84,24 @@ postsubmits:
       preset-service-account: "true"
       preset-bazel: "true"
     spec:
-      <<: *bazel_spec
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.4.10
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--clean"
+        - "--timeout=60"
```
Comment (rkpagadala) [inline prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml]: Piotr, this is the key line. Increase timeout to 60 minutes

File: prow/cluster/jobs/istio/proxy/istio.proxy.release-1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+presubmits:
+
+  istio/proxy:
+
+  - name: proxy-presubmit
+    agent: kubernetes
+    context: prow/proxy-presubmit.sh
+    branches:
+    - release-1.0
```
Comment (rkpagadala) [inline prow/cluster/jobs/istio/proxy/istio.proxy.release-1.0.yaml]: Piotr, this is a key line, use the branch release-1.0

File: prow/cluster/jobs/istio/proxy/istio.proxy.release-1.0.yaml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    labels:
+      preset-service-account: "true"
+      preset-bazel: "true"
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.4.10
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--clean"
+        - "--timeout=60"
```
Comment (rkpagadala) [inline prow/cluster/jobs/istio/proxy/istio.proxy.release-1.0.yaml]: Piotr, this is the key line. Increase timeout to 60 minutes

File: prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml (hunk 1 above)
Comment (PiotrSikora) [reply prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml]: Any reason not to globally bump `*bazel_spec` to 60mins?

**Review by PiotrSikora — APPROVED** [review]

File: prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml (hunk 1 above)
Comment (rkpagadala) [reply prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml]: not entirely sure. will have to talk to Seb

Comment (chxchx) [reply prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml]: I second this approach. Shorter jobs are not affacted (run time unchanged) and super long jobs that exceeded 60 should have been killed anyway. Much cleaner solution than duplicating the entire spec just to change one line

Comment (rkpagadala) [reply prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml]: ok will do

**Review by PiotrSikora — APPROVED** [review]

**Pull request merged**